packages/terraform-modules-public/aws/testnet/modules/startup-scripts/run-proxy-node.sh

```diff
@@ -1,19 +1,15 @@
 #! /bin/bash
 CELO_IMAGE=${celo_image}
-NETWORK_ID=${celo_network_id}
 CELO_VALIDATOR_SIGNER_ADDRESS=${validator_signer_address}
 
 NODE_DIRECTORY=/home/ubuntu/celo-proxy-node
 
 mkdir $NODE_DIRECTORY
 cd $NODE_DIRECTORY
-docker run -v $PWD:/root/.celo --rm $CELO_IMAGE init /celo/genesis.json
 
-BOOTNODE_ENODES="$(docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes)"
 PROXY_ADDRESS=${proxy_address}
 echo -n '${proxy_private_key_password}' > .password
 echo -n '${proxy_private_key_file_contents}' > keystore/${proxy_private_key_filename}
-echo -n '${proxy_node_private_key}' > .nodeprivatekey
```

Comment:
I didn't see a reference to this in the latest docs, but would love another set of eyes to make sure this is no longer needed